qa-brh.planx-pla.net/manifests/hatchery/hatchery.json

```diff
@@ -11,7 +11,7 @@
   "sidecar": {
     "cpu-limit": "1.0",
     "memory-limit": "256Mi",
-    "image": "quay.io/cdis/ecs-ws-sidecar:feat_datamounting",
+    "image": "quay.io/cdis/ecs-ws-sidecar:2022.09",
```

Comment:
not sure if `feat_datamounting` image is needed for hatchery. @jmontmaxwell do you know about this?

Reply:
Looking at the ecs-ws-sidecar history it looks like feat_datamounting was merged in May2022 so I think we should be fine @aartivnkt

Reply:
👍